src/function/handler.ts

```diff
@@ -18,7 +18,7 @@ export interface BackgroundHandler<C extends HandlerContext = HandlerContext> {
   (event: HandlerEvent, context: C): void | Promise<void>
 }
 
-export type Handler = BaseHandler<HandlerResponse, HandlerContext>
+export type Handler = BaseHandler<HandlerResponse | Response, HandlerContext>
```

Comment:
Maybe we could add a docs comment like this one instead?

```suggestion
/* For Lambda Compatiblity Mode */
export type Handler = BaseHandler<HandlerResponse | Response, HandlerContext>
```